tests/Unit/Application/Regulation/Query/GetCifsIncidentsQueryHandlerTest.php

```diff
@@ -23,23 +23,23 @@
 
 final class GetCifsIncidentsQueryHandlerTest extends TestCase
 {
+    private $regulationOrderRecordRepository;
```

Comment:
Ce serait bien, dans nos tests units, de typer les objets qu'on mock

```suggestion
    private MockObject $regulationOrderRecordRepository;
```